sdk/nodejs/region.ts

```diff
@@ -27,6 +27,7 @@ export let USEast1Region: Region      = "us-east-1";
 export let USEast2Region: Region      = "us-east-2";
 export let USWest1Region: Region      = "us-west-1";
 export let USWest2Region: Region      = "us-west-2";
+export let MESouth1: Region           = "me-south-1";
```

Comment:
Consider keeping the alphabetic order of the regions

Reply:
Done!